packages/engine/Source/Scene/VoxelPrimitive.js

```diff
@@ -1154,14 +1170,91 @@ function updateVerticalExaggeration(primitive, frameState) {
     primitive._maxBounds,
     primitive._exaggeratedMaxBounds
   );
-  if (defined(frameState) && primitive.shape === VoxelShapeType.ELLIPSOID) {
+  if (!defined(frameState)) {
```

Comment:
Why is `frameState` optional? Even if some shapes don't use it, we can still pass it in all the time, unless there is some other part of the architecture that is prevent that.

Reply:
I was calling `updateVerticalExaggeration` from the constructor (via `initialize`), when the `frameState` was not available. But this does create unnecessary branching.

I updated the `initialize` method to directly set the initial values for the exaggerated bounds & matrix, so `updateVerticalExaggeration` always follows the same path now.